package.json

```diff
@@ -163,7 +163,7 @@
 			},
 			{
 				"path": "./google-listings-and-ads.zip",
-				"maxSize": "8.22 mB",
+				"maxSize": "14.2 mB",
```

Comment:
Related to https://github.com/woocommerce/google-listings-and-ads/pull/3053#discussion_r2275675464.

After upgrading `googleads/google-ads-php` to `dev-legacy-v31.0.0`, appending `21` to [this script](https://github.com/woocommerce/google-listings-and-ads/blob/3.3.1/composer.json#L63) should keep the zip file size from increasing so significantly.